src/modules/twap/utils/getTwapOrderStatus.ts

```diff
@@ -9,11 +9,13 @@ const AUTH_THRESHOLD = ms`1m`
 export function getTwapOrderStatus(
   order: TWAPOrderStruct,
   isExecuted: boolean,
-  executionDate: Date,
+  executionDateStr: string | null,
```

Comment:
why not passing the Date or null directly?

Reply:
Good point, thanks!
Fixed